src/core/utils/LockHashUtils.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { sha3_256 } from 'js-sha3';
+import { Convert } from '../format/Convert';
+
+/**
+ * Hash utilities for SecretLock hashing
+ */
+export class LockHashUtils {
```

Comment:
Should it be called `Hashes`?

Reply:
these methods are only for the SecretLock hashes. Don't want to confuse the user with general hashes staffs

Reply:
fair enough